### DISCUSSION TITLE
Firebase source

### DISCUSSION THREAD

**Discussion by hounaine** [microsoft/Power-Fx#20] [Q&A] [answered]
Is there any chance to connect, - even in the future, my power apps to my firebase datas?

**Comment by MikeStall** [comment, marked as answer]
For general connector support, file your suggestion over here: https://powerusers.microsoft.com/t5/Power-Apps-Ideas/idb-p/PowerAppsIdeas

If you're writing an app that hosts Power Fx directly, your app can always:
- call your data source and pass the values into power fx expressions
- register custom functions that work against your data source.
